easybuild/easyconfigs/a/Amber/Amber-12.21-AmberTools-13.26-ictce-4.1.13.eb

```diff
@@ -0,0 +1,32 @@
+#
+# author: Benjamin P. Roberts (University of Auckland)
+#
+# based on work by Marios Constantinou (University of Cyprus)
+#
+
+name = 'Amber'
+version = "12"
+
+easyblock = 'EB_Amber'
+
+homepage = 'http://ambermd.org/amber.html'
+description = """Amber (originally Assisted Model Building with Energy
+Refinement) is software for performing molecular dynamics and structure
+prediction."""
+
+toolchain = {'name': 'ictce', 'version': '4.1.13'}
+toolchainopts = {'openmp': False, 'usempi': True}
+onlytcmod = True
```

Comment:
Please clarify why this is set (with a comment in the easyconfig)?

It's rarely used, so I guess you have a good reason?